donate/templates/pages/base_page.html

```diff
@@ -1,3 +1,5 @@
 {% extends "pages/base_page_master.html" %}
 
-{# There are no overrides in this template, but alternative apps can template-overide this file (e.g. thunderbird) #}
+{% block extra_header_classes %}
```

Comment:
Updating the header to have the class of `header__fundraise-up` so we can target it and add the requested change from the PR feedback which was "show desktop header icon on tablet breakpoint and above"